src/config.yaml

```diff
@@ -39,6 +39,33 @@ cpuids:
           name: vme
           bit: 0
 
+  0x00000007:
+    name: "Structured Extened Flags"
+    data_type:
+      type: SubLeafBitField
+      leaves:
+        - eax:
```

Comment:
what does it mean for there to be 3 of each register?

Reply:
This is a list of maps. each map is a struct that has fields eax,ebx,ecx,edx that are also lists.

Reply:
in this case, it means there could be 3 sub-leaves.

Reply:
it is probably worth looking at the `BitField` structure that those end up becoming.